zipkin_opentracing/src/opentracing.cc

```diff
@@ -271,17 +282,27 @@ class OtSpan : public ot::Span {
 class OtTracer : public ot::Tracer,
                  public std::enable_shared_from_this<OtTracer> {
 public:
-  explicit OtTracer(TracerPtr &&tracer) : tracer_{std::move(tracer)} {}
+  explicit OtTracer(TracerPtr &&tracer) : tracer_{std::move(tracer)}, sampler_{std::move(new ProbabilisticSampler(1.0))} {}
```

Comment:
`std::move(new ProbabilisticSampler(1.0))` can be changed to `new ProbabilisticSampler(1.0)`